### PR TITLE
Follow-up to Gluster end-to-end example (PR#1796)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1796

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tnguyen-rh @tpoitras PTAL

File: install_config/storage_examples/gluster_example.adoc [hunk 1]
```diff
@@ -12,34 +12,42 @@ toc::[]
...
+persistent volume claims (PVCs), and using GlusterFS as persistent storage.
 
-== Installing the _glusterfs-fuse_ Package
-The _glusterfs-fuse_ library must be installed on all *schedulable* OpenShift nodes:
 
 [NOTE]
 ====
-Often the OpenShift master node is not used to run pod workloads, and, thus, is not included as a schedulable node.
+All `oc ...` commands are executed on the {product-title} master node.
 ====
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_example.adoc]: Is "master node" a thing? Should it be "master host"?

File: install_config/storage_examples/gluster_example.adoc [hunk 2]
```diff
@@ -12,34 +12,42 @@ toc::[]
...
 ----
-# on each node:
 # yum install -y glusterfs-fuse
 ----
+====
+
+[NOTE]
+====
+The {product-title} master node is often not used to run pod workloads and, thus, is not included as a schedulable node.
+====
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_example.adoc]: Same as above. Maybe it should be "all-in-one host"?

File: install_config/storage_examples/gluster_example.adoc [hunk 3]
```diff
@@ -83,15 +92,16 @@ Verify that the endpoints were created:
...
 gluster-endpoints   192.168.122.21:1,192.168.122.22:1   1m
 ----
 ====
 
+[[complete-example-using-gusterfs-creating-the-persistent-volume]]
 == Creating the Persistent Volume
-The persistent volume (PV) file is defined next before creating the PV object in
-OpenShift:
+Next, the persistent volume (PV) file is defined, before creating the PV object
+in {product-title}:
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_example.adoc]: I don't think the comma is needed here. Unless you change the sentence to:

"Next, before creating the PV object, define the persistent volume:"

**Comment by bfallonf** [comment]
@ahardin-rh A few nits, but otherwise, LGTM.

**Comment by ahardin-rh** [comment]
@bfallonf Great suggestions. Fixed! Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/storage_examples/gluster_example.html[Storage Examples -> Complete Example Using GlusterFS]
|New topic providing an end-to-end example of how to use an existing Gluster cluster as an {product-title} persistent store.
%

**Pull request merged**